examples/cndp-frr/docker/cnetfwd-graph.jsonc

```diff
@@ -0,0 +1,178 @@
+        }
+    },
+
+    // List of all lports to be used in the application
+    // An lport is defined by a netdev/queue ID pair, which is a socket containing a Rx/Tx ring pair.
+    // Each queue ID is assigned to a single socket or a socket is the lport defined by netdev/qid.
+    // Note: A netdev can be shared between lports as the qid is unique per lport
+    //       If netdev is not defined or empty then it must be a virtual interface and not
+    //       associated with a netdev/queue ID.
+    // key/val - (R) The 'key' is the logical name e.g. 'eth0:0', 'eth1:0', ... to be used by the
+    //               application to reference an lport. The 'val' object contains information about
+    //               each lport.
+    //    netdev        - (R) The netdev device to be used, the part before the colon
+    //                     must reflect the netdev name
+    //    pmd           - (R) All PMDs have a name i.e. 'net_af_xdp', 'ring', ...
+    //    qid           - (R) Is the queue id to use for this lport, defined by ethtool command line
+    //    umem          - (R) The UMEM assigned to this lport
+    //    region        - (O) UMEM region index value, default region 0
+    //    busy_poll     - (O) Enable busy polling support, true or false, default false
+    //    busy_polling  -     Same as above
+    //    busy_timeout  - (O) 1-65535 or 0 - use default value, values in milliseconds
+    //    busy_budget   - (O) 0xFFFF disabled, 0 use default, >0 budget value
+    //    inhibit_prog_load - (O) inhibit loading the BPF program if true, default false
+    //    force_wakeup  - (O) force TX wakeup calls for CVL NIC, default false
+    //    skb_mode      - (O) Enable XDP_FLAGS_SKB_MODE when creating af_xdp socket, forces copy mode, default false
+    //    xsk_pin_path  - (O) Path to pinned xsk map for this port
+    //    description   - (O) the description, 'desc' can be used as well
+    "lports": {
+        "eth0:0": {
+            "pmd": "net_af_xdp",
+            "qid": 0,
+            "umem": "umem0",
+            "region": 0,
+            "description": "LAN 0 port",
+            "xsk_pin_path": "/sys/fs/bpf/eth0/xsk_map"
+        },
+        "eth1:0": {
+            "pmd": "net_af_xdp",
+            "qid": 0,
+            "umem": "umem0",
+            "region": 1,
+            "description": "LAN 1 port",
+            "xsk_pin_path": "/sys/fs/bpf/eth1/xsk_map"
+        }
+    },
+
+    // (O) Define the lcore groups for each thread to run
+    //     Can be integers or a string for a range of lcores
+    //     e.g. [10], [10-14,16], [10-12, 14-15, 17-18, 20]
+    // Names of a lcore group and its lcores assigned to the group.
+    // The initial group is for the main thread of the application.
+    // The default group is special and is used if a thread if not assigned to a group.
+    "lcore-groups": {
+        "initial": [0],
+        "timer": [1],
+        "group0": [2],
+        "group1": [3],
```

Comment:
I do not see a group1 being used, do you want to remove it?

Reply:
yes I will cleanup